### PR TITLE
Execute edit

### PR DESCRIPTION
**Pull request by jtregde**
Edited the execute file. I think the names of the variables being extracted from the `get_pop_objs()` are correct now, but someone can check that I'm reading that correctly from the demographics.py file.

### CODE REVIEW

File: scripts/execute.py [hunk 1]
```diff
@@ -27,9 +32,23 @@
...
 start_year = 2013
 pop_graphs = False
-(omega_path_S, imm_rates_path, rho_s, omega_SS, surv_rates_S, g_n_path,
-    g_n_SS, omega_S_preTP) = demog.get_pop_objs(E, S, T, min_age, max_age, start_year, pop_graphs)
-imm_rates_SS = imm_rates_path[-1, :]
+# Get the immigration rates for every period
+imm_rates_SS = demog.get_imm_resid(E+S, min_age, max_age, pop_graphs)
+
+# Get population objects
+(omega_path_S, g_n_SS, omega_SS, surv_rates_S, surv_rates,
```
Comment (jdebacker) [inline scripts/execute.py]: @jtregde I think you have one too many arguments coming back here.  There is only one vector of survival rates for each age.

You can test this by trying to execute this script and seeing if you get an error here.

Comment (jtregde) [reply scripts/execute.py]: Ah, I see, I had that extra `surv_rates` in there, but was missing that it returns the `imm_rates_mat` as well. It appears there are 8 things being returned from the `get_pop_objs` function, right?

Comment (jdebacker) [reply scripts/execute.py]: @jtregde You are right - 8 objects returned.

**Comment by jdebacker** [comment]
@jtregde Merge in the latest changes to the master to this branch (and resolve any conflicts) then let me know when this PR is great to go.

**Comment by jtregde** [comment]
@jdebacker it looks like this one is ready to go

**Pull request merged**